### PR TITLE
Adds custom property file

### PR DESCRIPTION
**Pull request by shubhamparikh927**

### CODE REVIEW

File: src/main/resources/hapi-r4.properties [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+# This is the address that the FHIR server will report as its own address.
+# If this server will be deployed (for example) to an internet accessible
+# server, put the DNS name of that server here.
+#
+# Note that this is also the address that the hapi-fhir-testpage-overlay
+# (the web UI similar to the one at http://hapi.fhir.org) will use to
+# connect internally to the FHIR server, so this also needs to be a name
+# accessible from the server itself.
+server_address=http://localhost:8080/hapi-fhir-jpaserver/fhir/
```
Comment (boxwala) [inline src/main/resources/hapi-r4.properties:14]: This value will be different for each server - e.g., https://fhir4-dev.elimuinformatics.com

File: src/main/resources/hapi-r4.properties [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+# Allowed Bundle Types for persistence (defaults are: COLLECTION,DOCUMENT,MESSAGE)
+##################################################
+#allowed_bundle_types=COLLECTION,DOCUMENT,MESSAGE,TRANSACTION,TRANSACTIONRESPONSE,BATCH,BATCHRESPONSE,HISTORY,SEARCHSET
+
+##################################################
+# Subscriptions
+##################################################
+
+# Enable REST Hook Subscription Channel
+subscription.resthook.enabled=false
```
Comment (boxwala) [inline src/main/resources/hapi-r4.properties:139]: Do we want this to be true

**Review by boxwala — COMMENTED** [review]
Please see two comments I added

File: src/main/resources/hapi-r4.properties [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+allow_cascading_deletes=true
+allow_placeholder_references=true
+expunge_enabled=true
+persistence_unit_name=HAPI_PU
+logger.name=fhirtest.access
+logger.format=Path[${servletPath}] Source[${requestHeader.x-forwarded-for}] Operation[${operationType} ${operationName} ${idOrResourceName}] UA[${requestHeader.user-agent}] Params[${requestParameters}] ResponseEncoding[${responseEncodingNoDefault}]
+logger.error_format=ERROR - ${requestVerb} ${requestUrl}
+logger.log_exceptions=true
+datasource.driver=org.h2.Driver
+datasource.url=jdbc:h2:file:./target/database/h2
```
Comment (hankwallace) [inline src/main/resources/hapi-r4.properties:39]: Shouldn't we change this to be a MySQL connection?

**Review by hankwallace — COMMENTED** [review]
Please add a Jira ticket key to this as well.

File: src/main/resources/hapi-r4.properties (hunk 3 above)
Comment (shubhamparikh927) [reply src/main/resources/hapi-r4.properties:39]: @hankwallace this is the first commit,
I am going to push all the changes needed for this in the next commit.

File: src/main/resources/hapi-r4.properties (hunk 2 above)
Comment (shubhamparikh927) [reply src/main/resources/hapi-r4.properties:139]: I will be committing these changes in next commit,
This is the first commit for custom property file

**Review by boxwala — APPROVED** [review]

**Pull request merged**